### ISSUE TITLE
Lots of Republicans can come from Free Worlds space.

### ISSUE THREAD

**Issue by ixPost** [endless-sky/endless-sky#8986]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

As I was in Wei trying to disable a gunboat to complete "Electron Beam", there suddenly came a massive Republican fleet including a Carrier from Kopernius.

### Steps to Reproduce

1. Go thru the Free Worlds story line until you get the "Electron Beam" mission.
2. Go to Wei and you might find a fleet coming from the south.
3. If not you can load the game and do step number 1 again.

### Expected Behavior

Only small fleets come from the south.

### Operating System

Ubuntu 22.10

### Game Source

Other (please describe)

### Game Version

0.9.8 Got it from Ubuntu Software source: ubuntu-jammy-universe (deb)

**Comment by lumbar527** [comment]
The Republic/Navy fleet was probably returning from a battle? Or a recognizance mission? Maybe?

**Comment by ixPost** [comment]
> The Republic/Navy fleet was probably returning from a battle? Or a recognizance mission? Maybe?

No, I have never seen so many in Free World space. Besides that many could capture a system perhaps.

**Comment by TheOnlyKnight** [comment]
I'm pretty sure it's an engine limitation that fleet spawns don't discriminate hyperlanes - the _system_ has a spawn list with appropriate rates, but the hyperlanes are all identical for the purpose of bringing the fleet in.

**Comment by Pshy0** [comment]
Related to #8876.

Even if you fix the current issue by preventing ships from spawning as jumping from a system they cant spawn in, ships could still jump away to that system and then come back. So it's not just about where they spawn, but also about what path they take.

**Comment by ixPost** [comment]
> I think a fleet going down south and then persisting and coming back up is fine!

No it's not because you never see such a huge fleet in Free Worlds systems and anyway such a huge fleet could wreak havoc on Free Worlds systems, if not capture one.

**Comment by Ferociousfeind** [comment]
Grumpy... Fleets spawn only in systems they're specified to, and will either launch from a planet in-system or jump in from one of the linked systems (or JD in, where source isn't discernible, or simply be in flight when you get there if all else fails), it's fine for navy fleets to be coming up from FW space. The FW fleets _do_ have a foothold in their space, they won't just keel over and die

**Comment by Amazinite** [comment]
This could now be addressed by #6489.
